coap-core/src/main/java/com/mbed/coap/utils/Cache.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (C) 2011-2018 ARM Limited. All rights reserved.
```

Comment:
Should it be
```suggestion
 * Copyright (C) 2011-2021 ARM Limited. All rights reserved.
```
Pelion or ARM?

Reply:
Actually if I change this, the license check seems to fail. I prefer to update the Copyright in another PR, because it will cause a lot of changes.